src/Foundation/Enum.cs

```diff
@@ -1332,4 +1332,9 @@ public enum NSUrlSessionDelayedRequestDisposition : long {
 		UseNewRequest = 1,
 		Cancel = 2,
 	}
+
+	[Native]
+	public enum NSXpcConnectionOptions : ulong {
+		Privileged = (1 << 12)
```

Comment:
We try to add a comma at the end of enum values (so that the diff only shows added lines if more values are added later):

```suggestion
		Privileged = (1 << 12),
```